tests/example-empty-file/expected_results.json

```diff
@@ -1,4 +1,5 @@
 {
+  "version": 2,
```

Comment:
I believe I actually caught a bug here? For two test cases, the `run.sh` script would not emit the version property.